tests/acceptance/test_files/cache.bats

```diff
@@ -353,7 +349,9 @@ load "$LIB_BATS_SUPPORT/load.bash"
   steampipe service stop
 
   # verify that `content` and `new_content` are the same
-  assert_equal "$new_content" "$content"
+  run jd -f patch output1.json output2.json
+  diff=$($FILE_PATH/json_patch.sh $output)
```

Comment:
We don't need to run the json_patch here. `json-patch.sh` was specifically made to compare snapshot outputs, since snapshots contain fields like `start_time` and `end_time`, json_patch.sh was used to strip those out of the jd patch file. You will need those in tests which compare snapshots like in dashboard.bats.

All you need to do here is `run jd -f patch output1.json output2.json`. I believe jd patch returns a non-zero exit code when there is a difference, or else returns 0 exit-code.

```
run jd -f patch output1.json output2.json
assert_success
```